ksql-common/src/main/java/io/confluent/ksql/function/KsqlFunction.java

```diff
@@ -129,8 +135,25 @@ private KsqlFunction(
   }
 
 
-  public Schema getReturnType() {
-    return returnType;
+  public Schema getReturnType(final List<Schema> arguments) {
```

Comment:
I have renamed the field to `parameters` and what is passed in as `arguments`. At some point I might go through the codebase and make this change everywhere because it's a distinction that is confused often.

Reply:
I'm not sure of the different between arguments and parameters - both are the same to me.  So I'm not sure changing the name makes the code in `getReturnType` any more readable.  But if we're changing the name, lets change the name of the getter, constructor param and toString too!

Reply:
> A parameter is a variable in a method definition. When a method is called, the arguments are the data you pass into the method's parameters.

https://stackoverflow.com/a/156787/2258040

I will make the corresponding changes.